x-pack/plugins/apm/ftr_e2e/cypress/integration/power_user/integration_settings/integration_policy.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+const integrationsPoliciesPath = '/app/integrations/detail/apm/policies';
+const policyName = 'apm-integration';
+const description = 'integration description';
+const host = 'myhost:8200';
+const url = 'http://myhost:8200';
+
+describe('when navigating to integration page', () => {
+  beforeEach(() => {
+    const integrationsPath = '/app/integrations/browse';
+
+    cy.loginAsPowerUser();
+    cy.visit(integrationsPath);
+  });
+
+  describe('creating a new integration policy', () => {
+    beforeEach(() => {
+      cy.get('[data-test-subj="integration-card:epr:apm:featured').click();
+      cy.get('[aria-selected="true"]').contains('Elastic APM in Fleet');
+      cy.contains('Elastic APM now available in Fleet!');
+      cy.contains('a', 'APM integration').click();
+      cy.url().should('include', 'app/integrations/detail/apm/overview');
+      cy.get('[data-test-subj="addIntegrationPolicyButton"]')
+        .should('not.disabled')
+        .click();
+
+      cy.get('[data-test-subj="packagePolicyDescriptionInput')
+        .clear()
+        .type(description);
+      cy.get('[data-test-subj="packagePolicyNameInput')
+        .clear()
+        .type(policyName);
+      cy.get('[data-test-subj="packagePolicyHostInput').clear().type(host);
+      cy.get('[data-test-subj="packagePolicyUrlInput').clear().type(url);
+    });
+
+    it('checks validators for required fields', () => {
```

Comment:
I checked only the required fields here. I think I should write different tests for the form itself. 
Let me know your thoughts